sympy/stats/rv.py

```diff
@@ -1076,57 +1081,73 @@ def sample(expr, condition=None, size=(), library='scipy',
         No modifications to environment's global seed settings
         are done by this argument.
 
+    Returns
+    =======
+
+    sample: float/list/numpy.ndarray
+        one sample or a collection of samples of the random expression.
+
+        - sample(X) returns float/numpy.float64/numpy.int64 object.
+        - sample(X, size=int/tuple) returns numpy.ndarray object.
+
     Examples
     ========
 
     >>> from sympy.stats import Die, sample, Normal, Geometric
     >>> X, Y, Z = Die('X', 6), Die('Y', 6), Die('Z', 6) # Finite Random Variable
-
-    >>> die_roll = sample(X + Y + Z) # doctest: +SKIP
-    >>> next(die_roll) # doctest: +SKIP
-    6
+    >>> die_roll = sample(X + Y + Z)
+    >>> die_roll # doctest: +SKIP
+    3
     >>> N = Normal('N', 3, 4) # Continuous Random Variable
-    >>> samp = next(sample(N)) # doctest: +SKIP
-    >>> samp in N.pspace.domain.set # doctest: +SKIP
+    >>> samp = sample(N)
+    >>> samp in N.pspace.domain.set
     True
-    >>> samp = next(sample(N, N>0)) # doctest: +SKIP
-    >>> samp > 0 # doctest: +SKIP
+    >>> samp = sample(N, N>0)
+    >>> samp > 0
     True
-    >>> samp_list = next(sample(N, size=4)) # doctest: +SKIP
-    >>> [sam in N.pspace.domain.set for sam in samp_list] # doctest: +SKIP
+    >>> samp_list = sample(N, size=4)
+    >>> [sam in N.pspace.domain.set for sam in samp_list]
     [True, True, True, True]
+    >>> sample(N, size = (2,3)) # doctest: +SKIP
+    array([[5.42519758, 6.40207856, 4.94991743],
+       [1.85819627, 6.83403519, 1.9412172 ]])
     >>> G = Geometric('G', 0.5) # Discrete Random Variable
-    >>> samp_list = next(sample(G, size=3)) # doctest: +SKIP
+    >>> samp_list = sample(G, size=3)
     >>> samp_list # doctest: +SKIP
-    array([10,  4,  1])
-    >>> [sam in G.pspace.domain.set for sam in samp_list] # doctest: +SKIP
+    [1, 3, 2]
+    >>> [sam in G.pspace.domain.set for sam in samp_list]
     [True, True, True]
     >>> MN = Normal("MN", [3, 4], [[2, 1], [1, 2]]) # Joint Random Variable
-    >>> samp_list = next(sample(MN, size=4)) # doctest: +SKIP
+    >>> samp_list = sample(MN, size=4)
     >>> samp_list # doctest: +SKIP
-    array([[4.22564264, 3.23364418],
-           [3.41002011, 4.60090908],
-           [3.76151866, 4.77617143],
-           [4.71440865, 2.65714157]])
-    >>> [tuple(sam) in MN.pspace.domain.set for sam in samp_list] # doctest: +SKIP
+    [array([2.85768055, 3.38954165]),
+     array([4.11163337, 4.3176591 ]),
+     array([0.79115232, 1.63232916]),
+     array([4.01747268, 3.96716083])]
+    >>> [tuple(sam) in MN.pspace.domain.set for sam in samp_list]
     [True, True, True, True]
 
+    .. versionchanged:: 1.7.0
+        sample returns an iterator containing the samples instead of value.
```

Comment:
This should be in the past tense to indicate that it is no longer the case.

Reply:
Changed it to `sample used to return an iterator`